helm-charts/charts/kube-starrocks/charts/starrocks/templates/starrockscluster.yaml

```diff
@@ -434,8 +434,8 @@ spec:
       configMapName: {{template "starrockscluster.be.configmap.name" . }}
       resolveKey: be.conf
     {{- if or .Values.starrocksBeSpec.storageSpec.name .Values.starrocksBeSpec.emptyDirs .Values.starrocksBeSpec.hostPaths }}
-    {{- if .Values.starrocksBeSpec.storageSpec.name }}
     storageVolumes:
+    {{- if .Values.starrocksBeSpec.storageSpec.name }}
```

Comment:
I have reproduced the error you found. I believe this line is key to resolving the issue. The other indentations should be fine; the number of spaces we are currently indenting with is consistent with the spaces before {{.

Reply:
ok, I revert the nindent is fine